src/redist/targets/packaging/windows/clisdk/Init.cmd

```diff
@@ -3,3 +3,14 @@
 set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 set DOTNET_MULTILEVEL_LOOKUP=0
 set PATH=%~dp0;%PATH%
+
+if not "%PROCESSOR_ARCHITECTURE%"=="x86" goto :SetDotnetRoot_Wow
+if not "%PROCESSOR_ARCHITEW6432%"== "" goto :SetDotnetRoot_Wow
+
+:SetDotnetRoot
+set DOTNET_ROOT=%~dp0
```

Comment:
Can we just set both all the time?

Reply:
I didn't want to point x64 things at our x86 runtime.